python/src/DistributionImplementation.i

```diff
@@ -92,3 +92,11 @@ Distribution __rmul__(Scalar s)
 
 
 } }
+
+%pythoncode %{
+def __DistributionImplementation_repr_markdown(self):
+    """Get Markdown representation."""
+    return self.__repr_markdown__()
+
+DistributionImplementation._repr_markdown_ = __DistributionImplementation_repr_markdown
```

Comment:
why rename it ?

Reply:
This is for consistency with _repr_html_ from Python's point of view, in order to make it consistent with other `_repr_` functions in [IPython.display.html](https://ipython.readthedocs.io/en/stable/api/generated/IPython.display.html)

Reply:
whatever, use the same name everywhere so renames like this can be dropped

Reply:
Ok. I removed the unnecessary renames at 25a1b7f8d.